### ISSUE TITLE
User authorisation

### ISSUE THREAD

**Issue by Dinesh100K** [ManVTish/rails_cart#2]
integrating Devise gem

**Comment by ManVTish** [comment]
Please do share some resources and working, like quick notes. Thanks.

**Comment by Dinesh100K** [comment]
added `gem 'devise'`
ran following commands
```
bundle install
rails generate devise:install
rails generate devise User
rails generate devise:views
```

**Comment by ManVTish** [comment]
This issue was addressed through PR #7
Thanks Dinesh.
